samples/app-link-unfurling-in-share-to-teams/nodejs/assets/sample.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-link-unfurling-in-share-to-teams-nodejs",
+    "source": "officeDev",
+    "title": "App link unfurling in share to teams",
+    "shortDescription": "Demonstrating feature of link unfurling in share to teams.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-link-unfurling-in-share-to-teams/nodejs",
+    "longDescription": [
+      "Demonstrating feature of link unfurling in share to teams."
```

Comment:
"This sample shows the feature of link unfurling in share to teams."

Reply:
Changed